functions/_common/grapherRenderer.ts

```diff
@@ -178,10 +190,15 @@ export async function fetchUnparsedGrapherConfig(
         ? [env.GRAPHER_CONFIG_R2_BUCKET_PATH]
         : ["by-branch", env.CF_PAGES_BRANCH]
 
+    const directory =
+        identifier.type === "slug"
+            ? R2GrapherConfigDirectory.publishedGrapherBySlug
+            : R2GrapherConfigDirectory.byUUID
+
```

Comment:
A kingdom for `match()` :)

Reply:
I thought about pulling in ts-pattern but it felt a bit premature at this point :)